scripts/docker-compose.yaml

```diff
@@ -0,0 +1,100 @@
+version: "3"
+services:
+  oidc-controller:
+    build:
+      context: ../oidc-controller
+      dockerfile: Dockerfile
+    entrypoint: bash -c "alembic upgrade head && uvicorn api.main:app --reload --host 0.0.0.0 --port 5000 --log-level info"
+    depends_on:
+      oidc-controller-db:
+        condition: service_healthy
+      ngrok-oidc-controller:
+        condition: service_started
+    environment:
+      - POSTGRESQL_HOST=${POSTGRESQL_HOST}
+      - POSTGRESQL_USER=${POSTGRESQL_USER}
+      - POSTGRESQL_PASSWORD=${POSTGRESQL_PASSWORD}
+      - POSTGRESQL_DB=${POSTGRESQL_DB}
+      - ACAPY_ADMIN_URL=${ACAPY_ADMIN_URL}
+      - ACAPY_ADMIN_URL_API_KEY=${ACAPY_ADMIN_URL_API_KEY}
+      - ACAPY_WALLET_ID=${ACAPY_WALLET_ID}
+      - ACAPY_WALLET_KEY=${ACAPY_WALLET_KEY}
+      - NGROK_TUNNEL_HOST=http://ngrok-oidc-controller:4040
+      - ACAPY_NGROK_TUNNEL_HOST=${ACAPY_NGROK_TUNNEL_HOST}
+    ports:
+      - ${CONTROLLER_SERVICE_PORT}:5000
+    volumes:
+      - ../oidc-controller:/app:rw
+    networks:
+      - vc_auth
+
+  oidc-controller-db:
+    image: "postgres:12"
+    environment:
+      - POSTGRES_PASSWORD=${POSTGRESQL_PASSWORD}
+      - POST
+    extra_hosts:
+      - host.docker.internal:host-gateway
+    volumes:
+      - oidc-controller-db-data:/var/lib/postgresql/data
+      - ./db_init.sql:/docker-entrypoint-initdb.d/1-init.sql
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U postgres"]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+    networks:
+      - vc_auth
+
+  keycloak:
+    image: jboss/keycloak:16.0.0
+    ports:
+      - 8880:8080
+    volumes:
+      - ./keycloak/config:/tmp
+    environment:
+      DB_VENDOR: ${KEYCLOAK_DB_VENDOR}
+      DB_ADDR: ${KEYCLOAK_DB_ADDR}
+      DB_DATABASE: ${KEYCLOAK_DB_NAME}
+      DB_USER: ${KEYCLOAK_DB_USER}
+      DB_PASSWORD: ${KEYCLOAK_DB_PASSWORD}
+      KEYCLOAK_USER: ${KEYCLOAK_USER}
+      KEYCLOAK_PASSWORD: ${KEYCLOAK_PASSWORD}
+      KEYCLOAK_LOGLEVEL: ${KEYCLOAK_LOGLEVEL}
+      ROOT_LOGLEVEL: ${KEYCLOAK_ROOT_LOGLEVEL}
+      KEYCLOAK_IMPORT: ${KEYCLOAK_IMPORT}
+    depends_on:
+      - keycloak-db
+    networks:
+      - vc_auth
+
+  keycloak-db:
+    image: registry.access.redhat.com/rhscl/postgresql-10-rhel7:latest
```

Comment:
We may want to upgrade to a newer version of postgres - I would go with the "vanilla", non RHEL version personally (we can also do this separately).

Reply:
quickly changing that to `postgres:12` cause the import to fail, will look into that in the future.